ixmp4/conf/settingsmodel.py

```diff
@@ -46,7 +46,13 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
         super().__init__(*args, **kwargs)
 
         self.setup_directories()
-        self.configure_logging(self.mode)
+
+        ixmp4_logger = logging.getLogger("ixmp4")
+        if ixmp4_logger.level == logging.NOTSET:
+            ixmp4_logger.setLevel(logging.INFO)
+
+        if len(ixmp4_logger.handlers) == 0:
+            ixmp4_logger.addHandler(logging.StreamHandler())
```

Comment:
I think in this setup, you'd get log duplication again if the logs are propagated and you log at warning level or above since the root logger is set at WARNING.